### PR TITLE
v2.4.1 imagelist

### PR DESCRIPTION
**Pull request by croomes**
Also changes the format of the Openshift version compatibility due to a scanner failure.  Copied from a RH project.

### CODE REVIEW

**Comment by croomes** [comment]
OLM operator was failing to install due to node not ready, caused by https://github.com/kubernetes-sigs/kind/issues/2240.  Bumping to latest kind resolved.  Also bumped `helm/kind-action` and Operator SDK to latest versions.

File: .github/workflows/test-index-image.yml [hunk 1]
```diff
@@ -46,16 +46,22 @@ jobs:
...
+        run: |
+          for i in {1..60}; do
+            kubectl get nodes -o 'jsonpath="{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}"' | grep "Ready=True" \
+              && break || sleep 2
+          done
       - name: Install OLM
         run: |
-          RELEASE_VERSION=v1.0.1
-          sudo curl -Lo /usr/local/bin/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+          RELEASE_VERSION=v1.5.1
```
Comment (darkowlzz) [inline .github/workflows/test-index-image.yml:61]: Why not v1.8.1, the latest?

**Review by darkowlzz — APPROVED** [review]
Thanks for finding and fixing the kind issue.
LGTM!

File: .github/workflows/test-index-image.yml (hunk 1 above)
Comment (croomes) [reply .github/workflows/test-index-image.yml]: I'm not sure!  Now v1.8.1.

**Pull request merged**